src/app/src/advancedsetupdialog.cpp

```diff
@@ -0,0 +1,15 @@
+#include <format>
+#include <spdlog/spdlog.h>
+
+#include <QFileDialog>
```

Comment:
I think you need `QDialog` here, since you're using that instead of `QFileDialog`